overview.html

```diff
@@ -219,6 +220,44 @@ <h3 id="Bulk document CRU">Bulk document CRU</h3>
     </LI>
 </UL>
 
+<h3 id="Document revisions">Document revisions</h3>
+<P>
+    This library does not modify the Java objects passed to the
+    {@link com.cloudant.client.api.Database#save(Object)},
+    {@link com.cloudant.client.api.Database#post(Object)},
+    {@link com.cloudant.client.api.Database#update(Object)},
+    or {@link com.cloudant.client.api.Database#bulk(List)} methods. As such once the server write
+    completes the Java object(s) will be out of sync with the database server regarding the
+    document revision. For any of these methods the new document revision can be obtained from
+    the server's {@link com.cloudant.client.api.model.Response} as shown here.
+</P>
+<pre>
+    {@code
+    Foo foo = new Foo();
+    Response response = db.save(foo);
+    foo.setRevision(response.getRevision());
+    }
+</pre>
+<P>
+    Note that storing the revision from a server response after saving a document may be
+    insufficient for the Java object to have the latest revision if the server document is
+    subsequently updated.
+</P>
+<P>
+    Documents retrieved from the server, for example using the
+    {@link com.cloudant.client.api.Database#find(Class, String)} method, are deserialized into
+    objects that will contain the revision as long as the type being deserialized into has a
+    {@code _rev} field or an appropriate custom serializer to ensure that the {@code _rev} is mapped
+    to another field.
+</P>
+<P>
+    Using the correct revision is essential for the
+    {@link com.cloudant.client.api.Database#update(Object)} and
+    {@link com.cloudant.client.api.Database#remove(Object)} methods, as well as when performing bulk
+    updates or deletions via the {@link com.cloudant.client.api.Database#bulk(List)} method. If the
+    latest revision is not used then a conflict will result.
```

Comment:
This is ambiguous -- a conflict _response_ will be sent but a conflict isn't created.